Directory.Build.props

```diff
@@ -80,20 +80,20 @@
 
   <!-- The TFMs to build and test against. -->
   <PropertyGroup>
-    <NetCoreAppCurrentVersion>8.0</NetCoreAppCurrentVersion>
+    <NetCoreAppCurrentVersion>9.0</NetCoreAppCurrentVersion>
```

Comment:
This wasn't supposed to happen in this PR, right? @ViktorHofer

Reply:
Basically we had the option to either work around the fact that we weren't doing this with the EnablePreviewFeatures work or just doing it and getting it over with.

I decided to do it as I didn't see a reason to work around the EnablePreviewFeatures issue instead of just fixing it and I thought it would be a much easier change than it ended up being.

If we want this to be in a separate branch, I can branch off from here with those changes in a draft PR, revert them here, and do the `EnablePreviewFeatures` workaround.

Reply:
It is unlikely that we can change the target here in a single pass, the sdk typically takes months to adjust

Reply:
I think we're actually pretty close to updating the TFM. Looks like we need to update the bootstrapping scripts for the installer tests and a little bit of debugging of the WBT legs.

Next week I'll branch off the upgrading to another PR and do the quick workaround for the EnablePreviewFeatures attribute to unblock this PR.